spec/acceptance/suites/default/10_with_pam_and_proc_spec.rb

```diff
@@ -0,0 +1,65 @@
+require 'spec_helper_acceptance'
+
+test_name 'simp::gdm with gnome and pam and hidepid'
```

Comment:
if you removed gnome from the tests, shouldn't `gnome` be removed from `test_name` and the `describe` below?